docs/wiki/deployment/yara.md

```diff
@@ -0,0 +1,138 @@
+# Introduction
+
+I recently put [YARA](https://github.com/plusvic/yara) inside [osquery](https://osquery.io) and thought I would provide some details on how to use it. There are two YARA related tables in osquery, which serve very different purposes. The first table, called **yara_events**, uses osquery's pub-sub framework to monitor for filesystem changes and will execute YARA when a file change event fires. The second table, called **yara**, is an on-demand YARA scanning table.
+
+# Configuration
+
+The configuration for osquery is simple. Here is an example config (I've left out the non-relevant parts):
+
+```
+  // Description of the YARA feature.
+  "yara": {
+    "signatures": {
+      // Each key is an arbitrary group name to give the signatures listed
+      "sig_group_1": [ "/Users/wxs/sigs/foo.sig", "/Users/wxs/sigs/bar.sig" ],
```

Comment:
we should take this opportunity to set a precedent for where yara signatures should be installed on the filesystem. Maybe make all of these paths `/etc/osquery/yara/foo.sig` and such?